packages/jest-editor-support/src/types.js

```diff
@@ -0,0 +1,73 @@
+// @flow
```

Comment:
I have not, my flow setup for the repo isn't quite perfect yet. I actually didn't know about that syntax :D